csharp/src/Drivers/BigQuery/BigQueryConnection.cs

```diff
@@ -1014,6 +1032,10 @@ public override AdbcStatement CreateStatement()
                 {
                     options[keyValuePair.Key] = keyValuePair.Value;
                 }
+                if(keyValuePair.Key == BigQueryParameters.LargeResultsDestinationTable)
```

Comment:
nit: space before paren to match the other tests

Unrelated to this specific change, does the ADBC spec say anything about case-sensitivity of options? For ODBC/JDBC/ADO.NET, connection string keys are usually case-insensitive.

Reply:
resolved nit

Reply:
> ~it's up to the driver, but mostly we've been case sensitive so far

This seems like something we'd want to provide guidance around to improve standardization e.g. "case-sensitive but consistently lower-case keys"